src/sentry/api/endpoints/release_deploys.py

```diff
@@ -63,13 +63,23 @@ def get(self, request: Request, organization, version) -> Response:
         if not self.has_release_permission(request, organization, release):
             raise ResourceDoesNotExist
 
-        queryset = Deploy.objects.filter(organization_id=organization.id, release=release)
+        release_project_envs = ReleaseProjectEnvironment.objects.select_related("release").filter(
+            release__organization_id=organization.id,
+            release__version=version,
+        )
+
+        project_id = request.GET.get("project", None)
```

Comment:
please do not access projects directly. This is a security issue as we are not checking that the user does not have access to the project passed in.

use `self.get_projects` like in https://github.com/getsentry/sentry/blob/1a1656ac9978f06a757c9d0acdba67e4c60cd9b5/src/sentry/api/endpoints/organization_group_index_stats.py#L74

Reply:
@JoshFerge thanks for the heads up about this helper function! How's it look now?